### PR TITLE
Feature/[ios] Write characteristics

### LINKED ISSUE #129: Read/write characteristics
Read/Write characteristics from following levels:

Device
Service
Characteristic

### PR DESCRIPTION
**Pull request by TomoLV**
Resolves #129

### CODE REVIEW

File: lib/internal/bridge/characteristics_mixin.dart [hunk 1]
```diff
@@ -153,7 +153,7 @@ mixin CharacteristicsMixin on FlutterBLE {
         .receiveBroadcastStream()
         .map(
           (rawJsonValue) =>
-              _parseCharacteristicResponse(peripheral, rawJsonValue),
+              _parseCharacteristicResponseWithValue(peripheral, rawJsonValue),
```
Comment (mikolak) [inline lib/internal/bridge/characteristics_mixin.dart]: `_parseCharacteristicWithValueResponse`? CharacteristicWithValue is a Dart type we've created.

Comment (TomoLV) [reply lib/internal/bridge/characteristics_mixin.dart]: Renamed to _parseCharacteristicWithValueResponse

**Comment by mikolak** [comment]
Conflicts

File: ios/Classes/FlutterBleLibPlugin.m [hunk 2]
```diff
@@ -205,25 +211,55 @@ - (void)readCharacteristicForDevice:(FlutterMethodCall *)call result:(FlutterRes
...
+                                   resolve:[self resolveForReadWriteCharacteristic:result]
+                                    reject:[self rejectForFlutterResult:result]];
+}
+
+- (void)writeCharacteristicForService:(FlutterMethodCall *)call result:(FlutterResult)result {
+    [_manager writeCharacteristicForService:[call.arguments[ARGUMENT_KEY_SERVICE_ID] doubleValue]
+                         characteristicUUID:call.arguments[ARGUMENT_KEY_CHARACTERISTIC_UUID]
+                                valueBase64:[self base64encodedStringFromBytes:call.arguments[ARGUMENT_KEY_BYTES]]
+                                   response:(BOOL)call.arguments[ARGUMENT_KEY_WITH_RESPONSE]
+                              transactionId:[ArgumentValidator validStringOrNil:call.arguments[ARGUMENT_KEY_TRANSACTION_ID]]
```
Comment (mikolak) [inline ios/Classes/FlutterBleLibPlugin.m]: We're not validating the value of this argument, this is only used to cast from something to String. Perhaps it should be renamed, both the call and the class?

Comment (mikolak) [reply ios/Classes/FlutterBleLibPlugin.m]: Create an issue and merge.

Comment (TomoLV) [reply ios/Classes/FlutterBleLibPlugin.m]: Issue created #181

**Pull request merged**